app/authentication/bearer_auth.py

```diff
@@ -7,6 +7,6 @@ class BearerAuth(requests.auth.AuthBase):
     def __init__(self, token: str):
         self.token = token
 
-    def __call__(self, r: requests.Request):
+    def __call__(self, r):
```

Comment:
just curious why a type annotation was removed

Reply:
`requests.Request` was not the correct type - `requests.auth.AuthBase` required a parent class of `requests.Request` for the type